plugins/out_gelf/gelf.c

```diff
@@ -460,6 +451,33 @@ static int cb_gelf_exit(void *data, struct flb_config *config)
     return 0;
 }
 
+/* Configuration properties map */
+static struct flb_config_map config_map[] = {
+    {
+     FLB_CONFIG_MAP_INT, "pckt_size", '1420',
```

Comment:
this property name is not right, you are using the structure field name instead of the property name that the plugin aims to read from the configuration